### DISCUSSION TITLE
Stellarium 23.1 install

### DISCUSSION THREAD

**Discussion by vidalv** [Stellarium/stellarium#3285] [General]
Hello all, after download from _**https://stellarium.org/fr/**_ for Windows 64 Bits Windows 7+ and run the file _**stellarium-23.1-qt5-win64.exe**_ I'm surprising the installation folder is under _**C:\Program Files (x86)**_, instead of **_C:\Program Files_** ! That mean the 64 Bits and 32 Bits are the same ? Thanks to clarify.
PS: Is there a X64 portable version ?
Regards.
Vincent.

**Comment by gzotti** [comment]
A little config error in the installer script. You can simply edit the instalation path to install to 'Program Files'.

We do not offer portable versions, but your search engine will likely find other sites for that.

**Comment by vidalv** [comment]
Thanks for clarification @gzotti  I'll do that for installation path !
Ok, no official portable version.
Thanks again.
